README.md

```diff
@@ -39,24 +39,28 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 <a name="prerequisites"></a>
 ### Prerequisites
 
-Minishift requires a hypervisor to run the virtual machine containing OpenShift. Depending on your
-host OS, you have the choice of the following hypervisors:
+Minishift requires a hypervisor to run the virtual machine containing OpenShift.
+Make sure that the hypervisor is installed and enabled on your system before you
```

Comment:
"the hypervisor" - which one? You are saying "the", but from the context it is not clear what this means. "Make sure that the hypervisor of your choice, ..."